core-primitives/top-pool/src/validated_pool.rs

```diff
@@ -44,6 +44,7 @@ use ita_stf::{ShardIdentifier, TrustedOperation as StfTrustedOperation};
 use itc_direct_rpc_server::SendRpcResponse;
 use itp_types::BlockHash as SidechainBlockHash;
 use jsonrpc_core::futures::channel::mpsc::{channel, Sender};
+#[allow(deprecated)]
 use retain_mut::RetainMut;
```

Comment:
Clippy suddenly went off about `RetainMut` being deprecated. Updated the library, and it seems to be fine now.

Either way, because of that Clippy lint I noticed two `unwrap()` below, and removed them accordingly.